src/server/cards/cornucopia/youngWitch.js

```diff
@@ -13,10 +13,24 @@ export default class YoungWitch extends Card {
     	if (await other.handleOwnReactions('attack', player, this)) {
         	return;
       	}
-      	const [card] = await other.selectCards({min:0, max:1, predicate: c => c.bane === true, message: 'Choose to reveal a bane card or not'});
-      	if (typeof card === 'undefined') {
+      	const [card] = await other.selectCards({min:0, max:1, predicate: c => c.title === this.bane, message: 'Choose to reveal a bane card or not'});
```

Comment:
`this` refers to different things in a `static` function than in a non-`static` function.  in `setup`, `this` is the `YoungWitch` class, but in `onPlay`, `this` is a particular card

Reply:
so `this.bane` will be undefined here

Reply:
also setting a the `bane` property on the `YoungWitch` class itself will make it so that if you have multiple games running at the same time, they will all have the same bane

Reply:
how about we check Card.classes.get('YoungWitch').bane and just ignore the multiple games issue...

Reply:
I also tested the above code, it worked even with this.bane